README.md

```diff
@@ -106,7 +106,7 @@ Current community packages are hosted on https://packagecloud.io/StackStorm. For
 
 If you are adding support for a new distribution for which ``packagingbuild`` and ``packagingtest``
 images are not yet published to Docker Hub and you want to test the build pipeline locally, you
-need to update ``docker-compose.yml`` file to use locally built Docker images.
+need to update ``docker compose.yml`` file to use locally built Docker images.
```

Comment:
``docker-compose.yml``

Reply:
Oh good spotting.  The entire sentence should be re-written.  docker-compose is v1 and is no longer functional with circle ci deprecating docker <1.20.  The docker compose plugin is required for v2 support, which available from the docker repository.